odml/terminology.py

```diff
@@ -40,9 +41,12 @@ def cache_load(url):
             if not os.path.exists(cache_dir):
                 raise
     cache_file = os.path.join(cache_dir, filename)
+    if replace_file and os.path.exists(cache_file):
```

Comment:
The whole clause and file remove is not required, since the file-write below (`file_obj = open(cache_file, "w")`) will truncate any existing content.